client/layout/guided-tours/tours/checklist-about-page-tour.js

```diff
@@ -21,6 +21,9 @@ import {
 	Tour,
 } from 'layout/guided-tours/config-elements';
 
+const SET_FEATURED_IMAGE_BUTTON_LABEL = translate( 'Set Featured Image' );
+const UPDATE_BUTTON_LABEL = translate( 'Update' );
```

Comment:
@akirk @lsinger It's not a good idea to translate strings statically on module initialization.

These function calls make the app initial load slower. Doing it once is negligible, but hundreds of calls are not and show up in profiles quite prominently.

l10n files are also loaded async -- initial render is always EN and gets rerendered when the l10n file request is done. For static calls like this, there is no mechanism to update the strings and they remain EN forever.

One way how to fix in is to wrap the actual call inside a lambda:
```js
const UpdateButton = localize( props => (
  <strong>{ props.translate( 'Update' ) }</strong>
) );

translate( 'Press the {{UpdateButton}} button', {
  components: { UpdateButton }
} );
```

I'll fix this in #22862 if nobody beats me to it with an independent PR.